src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -111,15 +182,14 @@ def main():
 
     # optional output
     rmedian = options["rmedian"]
+    vcategories = options["vcategories"]
     vmedian = options["vmedian"]
     transects = options["transects"]
     itransects = options["itransects"]
     csv_file_out = options["csvfilename"]
     resolutiondir = options["resolutiondir"]
 
-    overwrite_flag = ""
-    if gs.overwrite():
-        overwrite_flag = "t"
+    gs.overwrite()
```

Comment:
This is not doing anything right now. But you probably need to remove any other occurrences of `overwrite_flag`. Generally you don't need to set overwrite explicitly, the calls of a tool inside an addon will behave based on whether the addon was called with or without --o.